code/game/mecha/medical/odysseus.dm

```diff
@@ -9,25 +9,24 @@
 	internal_damage_threshold = 35
 	deflect_chance = 15
 	step_energy_drain = 6
-	var/builtin_hud_user = 0
+	var/builtin_hud_user = FALSE
 
 /obj/mecha/medical/odysseus/moved_inside(mob/living/carbon/human/H)
-	if(..())
+	. = ..()
+	if(.)
 		if(H.glasses && istype(H.glasses, /obj/item/clothing/glasses/hud))
-			occupant_message("<span class='warning'>Your [H.glasses] prevent you from using the built-in medical hud.</span>")
+			var/obj/item/clothing/glasses/hud/G = H.glasses
+			if (G.hud_type != DATA_HUD_MEDICAL_ADVANCED)
+				occupant_message("<span class='warning'>[H.glasses] prevent you from using the built-in medical HUD.</span>")

```

Comment:
honestly this makes no fucking sense since none of the existing huds actually conflict

Reply:
A relic from when they did.

Reply:
relics should be fixed properly.

Reply:
Would you prefer to just let the glasses and the mech's HUD both apply, then?

Reply:
I'd REALLY prefer if you weren't able to access security records while in a Odysseus mech. With the syringe gun available, its going to become a combat mech.

Reply:
I don't think it's a balance concern big enough to justify the lack of consistency. Just add them up, and make sure it doesn't remove medical HUDs on exit imo.